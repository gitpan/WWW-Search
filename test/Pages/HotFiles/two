http://www.hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/search.html?Utext=Medicine&Usrt=rel&Usrchtype=simple&search_max=26&
http://www.hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/search.html?Utext=Medicine&mainnext=23&Usrt=rel&Usrchtype=simple&search_max=26&
