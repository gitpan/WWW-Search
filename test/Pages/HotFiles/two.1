
<!-- Texis RDBMS Copyright (c) 1992 - 1998 Thunderstone -->
<!-- Texis Web Script Copyright (c) 1996 - 1998 Thunderstone -->



  <!-- 
       Copyright © 1999 ZDNet.  All rights reserved.
       Reproduction in whole or in part in any form or medium without express written permission of ZDNet is prohibited.
       ZDNet and the ZDNet logo are trademarks of Ziff-Davis Inc.
  -->
  

  <META NAME="WEB_COPY_DATE" CONTENT="19971010">
  <META NAME="WEB_CONTENT_TYPE" CONTENT="REVIEW">
  <META NAME="WEB_PAGE_CLASS" CONTENT="FILTER">
  
<HTML>
<HEAD>
<TITLE>Lycos Download Library - Search Results</TITLE>
</HEAD>

<BODY BGCOLOR=#FFFFFF>

<CENTER>
<FORM ACTION="http://www.lycos.com/cgi-bin/pursuit" METHOD=GET>
<INPUT TYPE=hidden NAME=matchmode VALUE=and>
<TABLE BORDER=0 BGCOLOR="#000000" CELLPADDING=0 CELLSPACING=0 WIDTH=600>
<TR>
<TD WIDTH=140 ALIGN=LEFT ROWSPAN=3><A HREF="http://www.lycos.com"><IMG SRC="http://lygo.com/ly/i/inv/lycoslogo.gif" HSPACE=12  WIDTH=116 HEIGHT=55 BORDER=0 ALT="Your Personal Internet Guide"></A></td>
<TD WIDTH=340 BGCOLOR="#000000" COLSPAN=3><IMG HEIGHT=1 WIDTH=340 SRC="http://lygo.com/ly/i/inv/dot_clear.gif"></TD>
<TD ROWSPAN=3 WIDTH=100><FONT FACE="VERDANA,HELVETICA,ARIAL" COLOR="#000000" SIZE=2>&nbsp;&nbsp;&nbsp;<A HREF="http://www.lycos.com/help/"><FONT COLOR="#FFFFFF">Help</FONT></A><BR>&nbsp;&nbsp;&nbsp;<A HREF="http://echomail.lycos.com/echomail"><FONT COLOR="#FFFFFF">Feedback</FONT></A></FONT></B></TD>
</TR>


<TR>
<TD ALIGN=RIGHT VALIGN="MIDDLE"><B><FONT size=-1 color="#FFCC00" FACE="VERDANA,HELVETICA,ARIAL">Search for:</FONT></TD>
<TD ALIGN=MIDDLE><INPUT TYPE=text NAME=query SIZE=20></TD>
<TD><INPUT TYPE=image src=http://lygo.com/ly/i/inv/gogetit455.gif BORDER=0></TD>
</TR>

<TR>
<TD COLSPAN=3><IMG HEIGHT=1 WIDTH=1 SRC="http://lygo.com/ly/i/inv/dot_clear.gif"></TD>
</TR></FORM>
</TABLE>
<TABLE WIDTH=600 BGCOLOR=#EEEEEE CELLSPACING=1 CELLPADDING=1 BORDER=0>
<TR><TD><FONT FACE=VERDANA,HELVETICA,ARIAL SIZE=-2><A HREF="http://www.lycos.com/network/"><FONT COLOR=#000000>Lycos Network</FONT></A>: <A HREF="http://www.lycos.com/cgi-bin/nph-bounce?nbv|http://www.tripod.com/"><FONT COLOR=#000000>Tripod.com</FONT></A> | <A HREF="http://www.lycos.com/cgi-bin/nph-bounce?nb_r|http://www.angelfire.com/"><FONT COLOR=#000000>Angelfire.com</FONT></A> | <A HREF="http://www.lycos.com/cgi-bin/nph-bounce?nb_r|http://mailcity.lycos.com/"><FONT COLOR=#000000>MailCity.com</FONT></A> | <A HREF="http://www.lycos.com/cgi-bin/nph-bounce?nb_r|http://www.whowhere.lycos.com/"><FONT COLOR=#000000>WhoWhere.com</FONT></A> | <A HREF="http://www.lycos.com/cgi-bin/nph-bounce?nb_r|http://www.hotbot.com/"><FONT COLOR=#000000>HotBot.com</FONT></A></FONT></TD><TD><FONT FACE=VERDANA,HELVETICA,ARIAL SIZE=-2><A HREF="http://www.lycos.com/cgi-bin/nph-bounce?nb_r|http://www.lycos.com/network/"><FONT COLOR=#000000>All Sites...</FONT></A></FONT></TD></TR></TABLE>

<P>
<FONT FACE="VERDANA,HELVETICA,ARIAL" COLOR="#000000" SIZE="-2"><!-- ad http://www.lycos.com/cgi-bin/aeapi?id=135&client_ip=147.81.3.197&service=ZDNET&uri=/search_zdnet&user-agent=libwww-perl/5.44 --> <CENTER><FONT SIZE=-2><a href="http://www.lycos.com/cgi-bin/nph-adclick.exe/CID=00065c2b8388a96800000000/ST=lycos/SV=ZDNET/A=.lycos.search_zdnet./T=1999.07.14.13.49.14"><img  src="http://ads07.focalink.com/SmartBanner/nph-graphic?198.1879-1999.07.14.13.49.14" alt="Get Compaq Quality PC's at Great Prices!!" border=0 WIDTH=468 HEIGHT=60></a><br><a href="http://www.lycos.com/cgi-bin/nph-adclick.exe/CID=00065c2b8388a96800000000/ST=lycos/SV=ZDNET/A=.lycos.search_zdnet./T=1999.07.14.13.49.14">Click here to visit site</a>
</FONT></CENTER>
<!--ea--><P>
<TABLE BORDER=0  WIDTH=600 CELLSPACING=0 CELLPADDING=0>
<TR><TD BGCOLOR="#EEEEEE" COLSPAN=4 HEIGHT=2 WIDTH=600>&nbsp;
<FONT FACE="VERDANA,HELVETICA,ARIAL" COLOR="#000000" SIZE="-1"><A HREF="http://www.lycos.com">Lycos Home</A>&nbsp;>&nbsp;<A HREF="http://www.lycos.com/computers/">Computers</A>&nbsp;>&nbsp;<a href="http://www.lycos.com/computers/miniguide/zdnet/downloads/index.html">Download Guide</a>&nbsp;>
</TD></TR>
<TR><TD BGCOLOR="#CCCCCC" COLSPAN=4 HEIGHT=2 WIDTH=600><IMG SRC="http://lygo.com/ly/i/inv/dot_clear.gif" BORDER=0 WIDTH=1 HEIGHT=1></TD></TR>
<TR><TD VALIGN=TOP BGCOLOR="#FFFFFF"  WIDTH=140>
<BR>
<font face="VERDANA,HELVETICA,ARIAL" size=-1>
<a href="http://www.lycos.com/computers/internet/">Internet Web Guide</a><p>
<a href="http://www.lycos.com/computers/miniguide/zdnet/downloads/index.html">Lycos Download Guide</a><p>  
<a href="http://www.lycos.com/computers/news/">Technology News</a><p>
<a href="http://boards.lycos.com/cgi-bin/WebX?13@253.43sKaYbsdLT^0@.ee6b313">Internet Message Board</a><p>
<a href="http://boards.lycos.com/cgi-bin/WebX?14@@.ee6b315">Software Message Board</a><p>
<a href="http://www.lycos.com/computers/miniguide/chat.html">Chat Online</a><p>
<a href="http://www.whowhere.lycos.com/Email">Find Email Addresses</a><p>
<a href="http://www.lycos.com/computers/books/bookstore.html">Computer Bookstore</a><p>
<a href="http://www.lycos.com/shopnet/software">Shop for Software</a><p>
<a href="http://auctionconnect.lycos.com/search?q=&c=29">Software Auctions</a><br>&nbsp;
</TD>      
<TD BGCOLOR="#CCCCCC" WIDTH=1 VALIGN=TOP><IMG SRC="http://lygo.com/ly/i/inv/dot_clear.gif" BORDER=0 WIDTH=1 HEIGHT=10></TD><TD WIDTH=13></TD> 
<TD WIDTH=446 VALIGN=TOP> &nbsp;<BR>

<TABLE BORDER=0><TR><TD VALIGN=ABSMIDDLE><IMG SRC="http://www.lycos.com/assist/graphics/computers-icon.gif" BORDER=0 WIDTH=26 HEIGHT=26></TD><TD WIDTH=10></TD><TD VALIGN=CENTER>
<FONT FACE="VERDANA,HELVETICA,ARIAL"><B>Software Downloads</B></FONT></TD></TR></TABLE>


<FORM METHOD="post" ACTION="http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/search.html">


<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0><TR>
<TD VALIGN=center><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=8 HEIGH=1 BORDER=0></TD>
<TD VALIGN=top ALIGN=left>
<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 WIDTH=100%><TR>
<TD VALIGN=center ALIGN=left>
<INPUT TYPE="HIDDEN" NAME="Usrt" VALUE="rel">
<INPUT TYPE="HIDDEN" NAME="Usrchtype" VALUE="simple">
<IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=20 HEIGHT=1 BORDER=0><FONT SIZE=3><INPUT TYPE="TEXT" NAME="Utext" SIZE=19 MAXLENGTH=30></FONT>
<INPUT TYPE="image" VALUE="Start Search" NAME="search" SRC="http://www.zdnet.com/swlib/images/sbutsse2.gif" BORDER=0 WIDTH=52 HEIGHT=16><p>
</FORM>
</TD><TD VALIGN=top ALIGN=right><A HREF="http://www.zdnet.com/chkpt/hud0005200/www.zdnet.com/swlib/"><IMG SRC="http://www.zdnet.com/swlib/images/zdnet_swl3.gif" WIDTH=147 HEIGHT=31 BORDER=0 ALT="ZDNet Software Library"></A><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=14 HEIGHT=1 BORDER=0></TD>
</TD>
</TR></TABLE><BR>





<TABLE BORDER=0 CELLSPACING=0 WIDTH=454><TR BGCOLOR="#CDEDFF">
<TD></TD>
<TD COLSPAN=2><FONT SIZE=2 COLOR="#000000" FACE="Arial"><b>Title/Synopsis<IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=1 HEIGHT=20 BORDER=0 ALIGN=absmiddle></TD>
<TD NOWRAP COLSPAN=2><FONT SIZE=2 COLOR="#000000" FACE="Arial"><b>Date</TD>
<TD NOWRAP COLSPAN=2><FONT SIZE=2 COLOR="#000000" FACE="Arial"><b>Rating</TD>
<TD NOWRAP COLSPAN=2><FONT SIZE=2 COLOR="#000000" FACE="Arial"><b>OS</TD>
<TD NOWRAP COLSPAN=1><FONT SIZE=2 COLOR="#000000" FACE="Arial"><b>Downloads</TD>
</TR>
<TR BGCOLOR="#FFFFFF">
<TD VALIGN=top BGCOLOR="#FFFFFF"></TD>
<TD ALIGN=left><FONT SIZE=2><b><A HREF="http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000NRQ">Twist and Shout BSP for Quake</b></FONT></A><BR>
<FONT SIZE=2>Play a Quake map with a track</FONT><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=1 HEIGHT=20 BORDER=0 ALIGN=top></TD>
<TD><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=6 HEIGHT=1 BORDER=0></TD>
<TD NOWRAP ALIGN=left><FONT SIZE=2>01-23-98</FONT></TD>
<TD><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=6 HEIGHT=1 BORDER=0></TD>
<TD ALIGN=left><IMG SRC="http://www.zdnet.com/swlib/images/nr.gif" WIDTH=58 HEIGHT=10 BORDER=0 ALT="Not Rated"></TD>
<TD><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=6 HEIGHT=1 BORDER=0></TD>
<TD ALIGN=left><FONT SIZE=2>DOS 5.0
</FONT></TD>
<TD><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=6 HEIGHT=1 BORDER=0></TD>
<TD ALIGN=left><FONT SIZE=2>51</FONT></TD>
</TR>
<TR BGCOLOR="#CDEDFF">
<TD VALIGN=top BGCOLOR="#FFFFFF"></TD>
<TD ALIGN=left><FONT SIZE=2><b><A HREF="http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000M0F">Subject Search Spider v1.16</b></FONT></A><BR>
<FONT SIZE=2>Make your web time more productive</FONT><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=1 HEIGHT=20 BORDER=0 ALIGN=top></TD>
<TD>&nbsp;</TD>
<TD NOWRAP ALIGN=left><FONT SIZE=2>04-20-99</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><IMG SRC="http://www.zdnet.com/swlib/images/4star.gif" WIDTH=58 HEIGHT=10 BORDER=0 ALT="4 stars"></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>Win95, 98, or NT
</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>10,472</FONT></TD>
</TR>
<TR BGCOLOR="#FFFFFF">
<TD VALIGN=top BGCOLOR="#FFFFFF"></TD>
<TD ALIGN=left><FONT SIZE=2><b><A HREF="http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000Q0B">NaturBase v1.33</b></FONT></A><BR>
<FONT SIZE=2>Get info on nutritional supplements</FONT><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=1 HEIGHT=20 BORDER=0 ALIGN=top></TD>
<TD>&nbsp;</TD>
<TD NOWRAP ALIGN=left><FONT SIZE=2>04-21-98</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><IMG SRC="http://www.zdnet.com/swlib/images/4star.gif" WIDTH=58 HEIGHT=10 BORDER=0 ALT="4 stars"></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>Win95
</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>1,215</FONT></TD>
</TR>
<TR BGCOLOR="#CDEDFF">
<TD VALIGN=top BGCOLOR="#FFFFFF"></TD>
<TD ALIGN=left><FONT SIZE=2><b><A HREF="http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000M90">Genocide Demo v0.95</b></FONT></A><BR>
<FONT SIZE=2>Annihilate everyone</FONT><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=1 HEIGHT=20 BORDER=0 ALIGN=top></TD>
<TD>&nbsp;</TD>
<TD NOWRAP ALIGN=left><FONT SIZE=2>12-30-97</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><IMG SRC="http://www.zdnet.com/swlib/images/2star.gif" WIDTH=58 HEIGHT=10 BORDER=0 ALT="2 stars"></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>Win95
</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>1,267</FONT></TD>
</TR>
<TR BGCOLOR="#FFFFFF">
<TD VALIGN=top BGCOLOR="#FFFFFF"></TD>
<TD ALIGN=left><FONT SIZE=2><b><A HREF="http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000EY8">Net.Medic v1.2.2</b></FONT></A><BR>
<FONT SIZE=2>Diagnose and repair connection problems</FONT><IMG SRC="http://www.zdnet.com/swlib/images/spacer.gif" WIDTH=1 HEIGHT=20 BORDER=0 ALIGN=top></TD>
<TD>&nbsp;</TD>
<TD NOWRAP ALIGN=left><FONT SIZE=2>09-18-98</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><IMG SRC="http://www.zdnet.com/swlib/images/5star.gif" WIDTH=58 HEIGHT=10 BORDER=0 ALT="5 stars"></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>Win95
</FONT></TD>
<TD>&nbsp;</TD>
<TD ALIGN=left><FONT SIZE=2>96,599</FONT></TD>
</TR>
</table>
<P>
<TABLE BORDER=0 WIDTH=470 CELLSPACING=0 CELLPADDING=0><TR>
<TD ALIGN=center><H3>End of hits.</H3></TD>
</TR></TABLE>

</TD>

</TR></TABLE>

</TD></TR>
<TR><TD BGCOLOR="#CCCCCC" COLSPAN=4 HEIGHT=2 WIDTH=600><IMG SRC="http://lygo.com/ly/i/inv/dot_clear.gif" BORDER=0 WIDTH=1 HEIGHT=1></TD></TR>
<TR><TD  COLSPAN=4 VALIGN=TOP WIDTH=600>
<BR>
<CENTER>
<FONT FACE="VERDANA,HELVETICA,ARIAL" COLOR="#000000" SIZE="-2">
<A HREF="http://www.lycos.com/lycosinc/legal.html#copy">Copyright</A>
&copy; 1998 Lycos, Inc. All Rights Reserved.  Lycos<font size=-3><sup>&#174;</sup></font> is a<BR> registered trademark of Carnegie Mellon University.<BR>
<A HREF="http://www.lycos.com/privacy/">Our Privacy Vow</A>&#160;&#160;<A HREF="http://www.lycos.com/lycosinc/legal.html">Terms and Conditions</A>&#160;&#160;<A HREF="http://www.lycos.com/lycosinc/legal.html#advertising">Standard Advertising Terms and Conditions</A>
</FONT>
</CENTER>
</TD></TR>
</TABLE></CENTER>
</BODY>
</HTML>

