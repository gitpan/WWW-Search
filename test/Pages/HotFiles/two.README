This set of test-result pages was created on Tue Sep 28 10:46:08 1999
