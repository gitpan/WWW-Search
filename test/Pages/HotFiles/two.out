http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000Q9G
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000ENB
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0009RD
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0002GG
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000X1Y
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0012H8
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000NL0
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000ZXH
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=00117W
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000CJU
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000TOE
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0011BB
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000CRM
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000G7T
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0010XO
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000RCP
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000VWC
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000P64
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000HB6
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000RD0
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000CRJ
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000I0H
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0000CI
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000O9F
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000O9F
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=00013H
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000SEG
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0005WU
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000NRQ
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000M0F
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000Q0B
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=0012B0
http://hotfiles.lycos.com/cgi-bin/texis/swlib/lycos/info.html?fcode=000M90
